projects/ngx-auth-firebaseui/src/lib/components/ngx-auth-firebaseui-register/ngx-auth-firebaseui-register.component.ts

```diff
@@ -116,7 +125,9 @@ export class NgxAuthFirebaseuiRegisterComponent implements OnInit, OnDestroy {
     this.registerForm = this.formBuilder.group({
       name: ['', Validators.required],
       email: ['', [Validators.required, Validators.email]],
-      password: ['', Validators.required],
+      password: ['', [Validators.required,
+                     Validators.minLength(this.config.passwordMinLength), 
+                     Validators.maxLength(this.config.passwordMaxLength)]], //TODO add Validator password
```

Comment:
i would remove the TODO as the scope of the PR is the length validator.

Reply:
Removed. Thanks.